melodic/distribution.yaml

```diff
@@ -7193,6 +7193,28 @@ repositories:
       url: https://gitlab.com/InstitutMaupertuis/topics_rviz_plugin.git
       version: melodic
     status: maintained
+  toposens:
+    doc:
+      type: git
+      url: 'https://gitlab.com/toposens/ros-projects/ts-ros.git '
```

Comment:
This url is quoted with an extra space at the end which likely isn't going to work well.